src/plugins/requestAuthPlugin.js

```diff
@@ -107,17 +107,17 @@ export const applyRequestAuthorizer$ = requestAuthorizer$ => request => {
  * Get an anonymous code from the API that can be used to generate an oauth
  * access token
  *
- * @param {Object} config { OAUTH_AUTH_URL, oauth }
+ * @param {Object} server the Hapi app server instance
```

Comment:
Unrelated thought --- should we be distinguishing between `server.settings.app` and `server.app`? From the [hapi documentation](https://hapijs.com/api):

```
app - application-specific configuration which can later be accessed via server.settings.app. Note the difference between server.settings.app which is used to store static configuration values and server.app which is meant for storing run-time state. Defaults to {}.
```

Perhaps a follow on task if we decide that we should.